assets/svg/rocket.svg

```diff
@@ -0,0 +1,219 @@
+<svg width="148" height="123" viewBox="0 0 148 123" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Looking at the change in VRT, I think we need to update the dimensions to match the image to make this a transparent replacement. If that doesn't work, then we'll need to specify the dimensions in the `RocketSVG` attributes so that there is no visual change.

Reply:
I played around with image size, and it's not the size that's an issue, it's positioning relative to image boundary, looking to get that cleared up which should resolve the discrepancies.
<img width="501" alt="Screen Shot 2021-02-25 at 5 22 48 PM" src="https://user-images.githubusercontent.com/373931/109241435-3e58cb00-778e-11eb-9278-0d52f70b8c64.png">

Reply:
Updated both the image to have the same positioning and size relative to image boundaries, and also added a css rule to control the svg size.